stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -238,8 +239,11 @@ namespace {
 struct ConformanceState {
   ConcurrentReadableHashMap<ConformanceCacheEntry> Cache;
   ConcurrentReadableArray<ConformanceSection> SectionsToScan;
-  
+  bool scanSectionsBackwards;
+
   ConformanceState() {
+    scanSectionsBackwards = swift::runtime::bincompat::
```

Comment:
Might be able to get away with `bincompat::workaroundProtocolConformanceReverseIteration()`.

Reply:
Almost. `runtime::bincompat::...` works. Updated.